### ISSUE TITLE
Assets conflict during OCR text and pdf conversion

### ISSUE THREAD

**Issue by s-katte** [astrosonic/Image-Augmentation-OCR#9]
While one client is performing OCR, 'test.txt' and 'test.pdf' are the only files and this will create conflict when multiple clients are there.

**Comment by mohitgurav** [comment]
You are right. We can deal with this issue by creating a hash for each file uploaded by the user. Let's work on it.

**Comment by s-katte** [comment]
Sure, I will try to solve it
